doc/api/process.markdown

```diff
@@ -457,6 +583,91 @@ console.log(process.env.TEST);
 // => undefined
 ```
 
+## process.emitWarning(warning)
+
+The `process.emitWarning(warning)` method can be used to emit custom 
+or application specific process warnings. These can be listened for by adding
+a handler to the [`process.on('warning')`][process_warning] event.
+
+* `warning` {String|Object|Error} The warning to issue
+
+The `warning` argument can be either a string, an Error object, or a simple 
+JavaScript object with `name` and `message` properties.
+
+If `warning` is passed as a string, the warning name is set to `'Warning'` and
+the provided string is used as the message.
+
+```js
+// Emit a warning using a string...
+process.emitWarning('Warning! Something happened!');
+  // Emits: (node: 56338) Warning: Warning! Something happened!
```

Comment:
The first `Warning` in `Warning: Warning!` is the `name` property of the Error object that is emitted. The second comes from the message.